Formula/mtoc.rb

```diff
@@ -0,0 +1,39 @@
+class Mtoc < Formula
+  desc "Mach-O to PE/COFF binary converter"
+  homepage "https://opensource.apple.com/source/cctools/cctools-927.0.2/"
+  url "https://opensource.apple.com/tarballs/cctools/cctools-927.0.2.tar.gz"
+  sha256 "3d5d35e7120aac187de306019d4b53b3b08dd7ebf48764fcd5c8cefef40b3dbf"
+  depends_on "llvm" => :build
+
+  def install
+    build_include = "#{buildpath}/include"
+    cp_r "#{HOMEBREW_PREFIX}/opt/llvm/include/llvm", build_include
+    cp_r "#{HOMEBREW_PREFIX}/opt/llvm/include/llvm-c", build_include
+
+    # Conflicts with _structs.h in MacOSX10.15.sdk
+    File.delete "include/mach/i386/_structs.h" if MacOS.version == :catalina
```

Comment:
This should probably be fixed upstream instead of being hotfixed here.

Reply:
Seems like upstream may be internal Apple SDKs sadly....

Reply:
I'd be glad to send a fix upstream but there's no way to do that as @rickmark noticed.

Reply:
It could well be fixed in 949.0.1 that is ["there"](https://opensource.apple.com/tarballs/cctools/) but the link is broken.

Reply:
@Bo98, yeah I wasn't able to use the link as well. I mentioned that in the PR